### PR TITLE
Add additional roles on signup

### PR DESCRIPTION
**Pull request by Charl1996**
[Ticket](https://dimagi-dev.atlassian.net/browse/SC-2363)

Add a constant specifying additional roles for a user on top of the base "Gamma" role.

### CODE REVIEW

File: superset_config.py
Comment (sravfeyn) [inline superset_config.py:41]: Good idea, could name it as local_config to keep it consistent with flask/superset terminology.

File: superset_config.py
Comment (sravfeyn) [inline superset_config.py:37]: Is `Gamma` not a subset of `sql_lab`?

Comment (sravfeyn) [reply superset_config.py:37]: I don't know as well, but the behaviour seemed as if it's so. You can find that by comparing the permissions for each role under Edit Role where role permissions are listed in your local superset env.

Comment (SmittieC) [reply superset_config.py:37]: https://superset.apache.org/docs/security/#:~:text=The%20sql_lab%20role%20grants%20access%20to%20SQL%20Lab.%20Note%20that%20while%20Admin%20users%20have%20access%20to%20all%20databases%20by%20default%2C%20both%20Alpha%20and%20Gamma%20users%20need%20to%20be%20given%20access%20on%20a%20per%20database%20basis.

File: superset_config.py
Comment (Charl1996) [reply superset_config.py:41]: Agree!

File: superset_config.py
Comment (SmittieC) [inline superset_config.py:42]: I think we should catch `ModuleNotFound` explicitly here. Reason being that if there's some logic in the `local_config.py` module that is erroneous, it will be masked by this `except` clause, resulting in unexpected behaviour.

File: superset_config.py
Comment (sravfeyn) [reply superset_config.py:37]: Great, thanks for digging.

**Comment by sravfeyn** [comment]
Looks good now. This should be QAed on http://commcare-analytics-demo.dimagi.com/ before merging similar to https://dimagi-dev.atlassian.net/browse/QA-174 with any additional SQL lab related tests to make sure queries can't access other domains data.

**Comment by Charl1996** [comment]
@sravfeyn I just bumped the superset version on this PR (probably a good idea?)

**Comment by Charl1996** [comment]
Seems to work fine on the demo version.

**Comment by sravfeyn** [comment]
>  just bumped the superset version on this PR 

I presume you mean the hq_superset version. If that's so, we don't really follow versioning, neither is this is a proper pip package. So there's really no point. 

But it does feel very nice to bump up the versions 😆

**Comment by Charl1996** [comment]
@sravfeyn After testing again on the demo version it seems this works. Also, I can't access tables from other domains through SQL lab. Feel free to also verify if you like.

Could I perhaps get an approval if all is well?

**Review by sravfeyn — APPROVED** [review]
Great work, Charl, thank you!

**Pull request merged**
